### ISSUE TITLE
Moped Tables | Refine GridTable layout/design on Projects page

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#5028]
- [x] Change page title from "Projects List" to "Projects"
- [x] Make [fixed tabs](https://material-ui.com/components/tabs/#fixed-tabs) for "Search" (currently "General Search") and "Advanced Search" (currently "Filter Search") inside the search box.
- [x] Move "Download" button (see #4785) into search box. 

<img width="1351" alt="Projects-Page-Design-Refinements" src="https://user-images.githubusercontent.com/1463708/105315484-02c25400-5b85-11eb-8e73-5b7876b99b2e.png">

**Comment by amenity** [comment]
Current design:
<img width="1346" alt="Screen Shot 2021-01-21 at 1 14 01 AM" src="https://user-images.githubusercontent.com/1463708/105316093-00142e80-5b86-11eb-9b5b-d35bf36c21ad.png">

@johnclary @mateoclarke @zappfyllc lmk if you think these changes make sense. I don't love the way they look, but they make more sense from a functional/UX perspective to me. 

Also lmk if you think they're worth waiting on before closing this epic or belong in the backlog.

**Comment by johnclary** [comment]
Makes sense. Looks good!

**Comment by amenity** [comment]
On second thought, it probably makes sense for the "Download" button to be at the top of the project-list component box. Since that's what the user is downloading?

**Comment by amenity** [comment]
Not loving the clutter added by horizontal tabs... A "Filters" switch <https://material-ui.com/components/switches/> would look nicer but doesn't make sense, logically. 

Need to mock up this solution @johnclary offered via Slack: 

- [ ] Remove the field picker for normal search — default to search Name, Description, and Project ID
- [ ] Replace "Clear" button with "Add Filters" button

**Comment by sergiogcx** [comment]
There is a PR:
https://github.com/cityofaustin/atd-moped/pull/194

Preview URL:
https://deploy-preview-194--atd-moped-main.netlify.app/

**Comment by amenity** [comment]
Aww dang — sorry y'all, @sergiogcx @zappfyllc . 🤦🏻  @johnclary and I had discussed other changes to this, which is why I had it in "Needs Scoping".

**Comment by mateoclarke** [comment]
Should I hold off on reviewing @amenity?

**Comment by amenity** [comment]
Yes, @mateoclarke. 😞

**Comment by amenity** [comment]
-[ ] Change "Export CSV" to "Download"

Search
- [ ] Remove "Field" picker and default to searching _Project name_, _Project description_, and _eCAPRIS subproject ID_. 
- [ ] Update placeholder text "Search by name, description, or eCAPRIS subproject ID"
- [ ] Remove "Clear" button

Advanced Search
- [ ] Replace trash icon with "X"

**Comment by amenity** [comment]
@sergiogcx - this is ready now. Sorry again about the miscommunication!

**Comment by sergiogcx** [comment]
@amenity It's definitely possible, but it is a major refactor of the code, this is because the search component is meant to be re-usable, meaning all the changes we make to it will affect all other uses, (i.e., the users table).

In other words, it will take a bit of time to allocate for this configuration change.

**Comment by zappfyllc** [comment]
@amenity what if we close out "Moped Tables" with the generic changes that can happen across Moped Table components, but then create a separate "Search Projects" feature epic to contain the work on searching projects? I imagine as we change/add columns to this view, as well as discuss searching projects on the Map, we're going to want to hone our search functionality and I definitely think tackling that as separate work is a good idea based on my experience.

@sergiogcx I'm not sure if @amenity and @johnclary want this, but would a keyword search across all columns in that table be easier to implement for Moped Tables by and large? I know the pushback from a user perspective will be that we don't want certain columns/fields ruining the search because of sensitivity (i.e. picking up numbers or characters from other irrelevant columns), but I'm not certain that's worth designing around in a reusable feature, at least at the first pass...I'd rather see if it becomes a _common_ problem with users first and then address it as an improvement.

In my mind, "keyword search" could be a sub-feature of Moped Tables, while a more fine-tuned "project search" could be its own standalone feature for searching projects in Moped in a very targeted way but TBD on 1) whether that's needed and 2) what that algorithm or those search patterns will look like.

**Comment by amenity** [comment]
@zappfyllc @sergiogcx can definitely create a separate issue for follow up work! The only change that I’d really want to see in this commit would be getting rid of the “clear” button on simple search

**Comment by sergiogcx** [comment]
@amenity @zappfyllc As it turns out, it was a tough cookie to make the changes, but I was able to figure out a way. The PR should be fully functional and it includes those changes.

https://deploy-preview-194--atd-moped-main.netlify.app/
